polyglot/download.py

```diff
@@ -0,0 +1,250 @@
+import gzip
+
+from bitsv.network import NetworkAPI
+
+from .bitcom import B, C, BCAT, BCATPART, D, AIP, MAP
+
+class Download(NetworkAPI):
+    def __init__(self, network='main'):
+        super().__init__(network=network)
+
+    # UTILITIES
+    @staticmethod
+    def binary_to_file(binary, file):
+        """Give the pathname to a png, jpg, gif, tiff, bmp
+        writes binary to it
+
+        Example path (for newer users): "C://Users/username/Pictures/my_picture.jpg etc."
+        """
+        with open(file, 'wb') as f:
```

Comment:
I think this would be improved with this:

```
        os.makedirs(os.path.dirname(file), exist_ok=True)
        with open(file, 'wb') as f:
            f.write(binary)
```
Do you agree?

But I can also do this post merge to clear the backlog 😃

Reply:
I was just trying to guess-copy your existing code organization a little here.  Making the folders automatically sounds fine.  Might want to document it since it differs from the behaviors of most python functions.